nmoscommon/webapi.py

```diff
@@ -691,75 +752,77 @@ def error(self, e):
 
         bm = request.accept_mimetypes.best_match(['application/json', 'text/html'])
         if bm == 'text/html':
-            if isinstance(e,HTTPException):
+            if isinstance(e, HTTPException):
                 if e.code == 400:
-                    (t,v,tb) = sys.exc_info()
-                    return IppResponse(highlight('\n'.join(traceback.format_exception(t,v,tb)), PythonTracebackLexer(), HtmlFormatter(linenos='table',
-                                                                                                                                      full=True,
-                                                                                                                                      title="{}: {}".format(e.code, e.description))),
-                        status=e.code,
-                        mimetype='text/html')
+                    (exceptionType, value, traceback) = sys.exc_info()
```

Comment:
Slightly uncertain about using the word traceback here and importing traceback under a different name. It'll work, but I wonder if it might be more likely to cause errors when making changes in the future?

Could 'value' be improved at all?